docs/reference/commandline/swarm_init.md

```diff
@@ -142,6 +143,11 @@ Format example is `--default-addr-pool-mask-length 24`
 
 This flag sets up task history retention limit.
 
+### `--data-path-port`
+
+This flag specifies data path port number.
```

Comment:
This doesn't provide a lot of information beyond the flag name itself. Perhaps you can write a bit more about what the purpose is of this flag, and in which case you'd want to set this option. I guess we also want to mention;

- (as mentioned above) why set this option?
- The default (if this option is not set)
- That this is used for the whole swarm (i.e., it's not configurable _per node_)
- That it's an UDP port
- Valid port range
- That it can only be set when initializing the swam. Updating is not (yet?) supported

If you need help phrasing; can you reach out to @ahh-docker ?